restdocs-api-spec-jsonschema/src/main/kotlin/com/epages/restdocs/apispec/jsonschema/JsonSchemaFromFieldDescriptorsGenerator.kt

```diff
@@ -69,10 +69,13 @@ class JsonSchemaFromFieldDescriptorsGenerator {
         val objectMapper = jacksonObjectMapper().enable(SerializationFeature.INDENT_OUTPUT)
         return StringWriter().use {
             schema.describeTo(JSONPrinter(it))
-            objectMapper.writeValueAsString(objectMapper.readTree(it.toString()))
+            val rawJsonSchema = cleanupRawJsonSchema(it.toString())
```

Comment:
Could this be done on the schema object or a copy?

I see the risk of unintended string replacement, e.g. if someone uses "enum" by accident or as placeholder:
```
{
"description": "enum"
}
```

Reply:
I was trying out a lot of different things before introducing this hack :-) The issue is that the org.everit.json.schema.EventSchema does not set the type property when writing in the JSON stream. But that's the only place where the property are set... There is no simple way of setting or overriding some property on schema level (without using reflections) . But I will give it another try. Otherwise I will try to 'safer' hack then the one applied. I totally agree with the risk mentioned below.

Reply:
Found a non-hacky way :-)

How about that -> https://github.com/ePages-de/restdocs-api-spec/pull/130/commits/3321b62b4690ca9076fdf80174f6891e7f97994c

Reply:
What I was looking! :)